roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -179,16 +179,18 @@ resource "openstack_compute_instance_v2" "control" {
   EOF
 }
 
-resource "openstack_compute_instance_v2" "compute" {
-  count = {{ compute_count }}
+{% for partition in openhpc_slurm_partitions %} 
+resource "openstack_compute_instance_v2" "{{ partition.name }}" {
+  count = {{ partition.count }}
 
-  name      = "{{ cluster_name }}-compute-${count.index}"
+  name      = "{{ cluster_name }}-compute-{{ partition.name }}-${count.index}"
```

Comment:
It would be nice if this was de-duped - the default setup will produce nodes names like `<clustername>-compute-compute-N` right?

Reply:
Yeah, you're right. I actually don't mind having "compute" in there, because it distinguishes it from control/login. I'll make the change in `group_vars` and replace the partition name with "standard" instead.